app/page.tsx

```diff
@@ -176,7 +176,7 @@ function StatsCardBody() {
                     </td>
                 </tr>
             )}
-            {blockTime && (
+            {blockTime !== undefined && (
```

Comment:
<img width="640" alt="image" src="https://github.com/user-attachments/assets/ac7c9fb6-2e07-4e0c-b0a4-4f3cde2402ae" />
There was a random "0" because React evaluates `blockTime` to 0 and then displays that element. This should return the current time on the cluster, but we haven't implemented that yet.

Reply:
_Oh JavaScript, how beautiful you are _